monitoring/dashboards/dashboard.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/tidwall/gjson"
+	"github.com/tidwall/sjson"
+)
+
+const (
+	datasourceName = "dm-cluster"
+	outputDir      = "./dashboards"
+	name           = "dm.json"
+)
+
+var (
+	dashboards = map[string]string{
+		"dm.json": "Test-Cluster-DM",
+	}
+)
+
+func readDashboard(dir string, name string) (string, error) {
+	file := filepath.Join(dir, name)
+	data, err := ioutil.ReadFile(file)
+	if err != nil {
+		return "", errors.New(fmt.Sprintf("File %s do not exists", file))
+	}
+	return string(data), nil
+}
+
+func writeDashboard(dir string, name string, body string) error {
+	title, exist := dashboards[name]
+	if !exist {
+		return errors.New(fmt.Sprintf("%s dashboard is not found in operator", name))
```

Comment:
use fmt.Errorf or pingcap/errors instead

Reply:
This `errors.New` defines in pingcap/errors indeed.

Reply:
You can try `errors.Errorf`.